src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './App.css';
+import IndexNavbar from './components/IndexNavbar';
+import TabNav from "./components/TabNav";
+import {Container} from "react-bootstrap";
+import TabView from "./components/TabView";
+import {TabData, TableData} from "./api/Interfaces";
+
+const debugUrl = "http://localhost:8080";
+
+interface AppState {
+    columns: [],
+    tables: Array<TableData>,
+    tabs: Array<TabData>,
+    currentTab?: TabData
+}
+
+class App extends React.Component {
+    state: AppState = {
+        columns: [],
+        tables: Array<TableData>(),
+        tabs: Array<TabData>(),
+        currentTab: undefined
+    }
+
+    componentDidMount() {
+        fetch(debugUrl + '/api/columns')
+            .then(r => r.json())
+            .then(cols => {
+                this.setState({columns: cols});
+            })
+        fetch(debugUrl + '/api/tables')
+            .then(r => r.json())
+            .then(tables => {
+                this.setState({tables: tables});
+            })
+        fetch(debugUrl + '/api/tabs')
+            .then(r => r.json())
+            .then(tabs => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7256353)
```suggestion
            .then((tabs) => {
```